js/main.js

```diff
@@ -44,6 +48,131 @@ function getShowDetail(id) {
       genre: xhr.response.genre_names
     };
     console.log(showDetailObject);
+    $oneCard.append(renderSummary(showDetailObject));
+    goToSummaryPage(showDetailObject);
   });
   xhr.send();
 }
+
+function renderSummary(entry) {
+  var div1 = document.createElement('div');
```

Comment:
These numbered variables need better names. You should name them after the content they will hold.